ndm-exporter/collector/seachest.go

```diff
@@ -52,8 +52,23 @@ type SeachestCollector struct {
 // SeachestMetricData is the struct which holds the data from seachest library
 // corresponding to each blockdevice
 type SeachestMetricData struct {
-	SeachestIdentifier *seachest.Identifier
-	TempInfo           blockdevice.TemperatureInformation
+	SeachestIdentifier   *seachest.Identifier
+	TempInfo             blockdevice.TemperatureInformation
+	ModelNumber          string
+	SerialNumber         string
+	VendorID             string
+	FirmwareVersion      string
+	UUID                 string
+	Capacity             uint64
+	LogicalSectorSize    uint32
+	PhysicalSectorSize   uint32
```

Comment:
These fields are fixed for a drive and will not change. Also this information is available in the etcd. For pulling metrics from etcd we have the ndm-cluster-exporter and the static collector. 

These fields, and any other field that does not change in values / value is available in etcd should be fetched via the static collector. This reduces pressure on the disk, since each metrics collection involves some SCSI commands to the disk.